types/office-js-preview/index.d.ts

```diff
@@ -10189,9 +10189,9 @@ declare namespace Office {
         /**
          * Gets an attachment from a message or appointment and returns it as an `AttachmentContent` object.
          *
-         * The `getAttachmentContentAsync` method gets the attachment with the specified identifier from the item. As a best practice, you should use
-         * the identifier to retrieve an attachment in the same session that the attachmentIds were retrieved with the `getAttachmentsAsync` or
-         * `item.attachments` call. In Outlook on the web and mobile devices, the attachment identifier is valid only within the same session.
+         * The `getAttachmentContentAsync` method gets the attachment with the specified identifier from the item. As a best practice, you should use 
+         * the identifier to retrieve an attachment in the same session that the attachment ID was retrieved using the `getAttachmentsAsync` call.
```

Comment:
I wonder if we can say something like this
```suggestion
         * The `getAttachmentContentAsync` method gets the attachment with the specified identifier from the item. As a best practice, you should get
         * the attachment's identifier from a `getAttachmentsAsync` call then, in the same session, use that identifier to retrieve the attachment.
```